python/ray/autoscaler/_private/aws/node_provider.py

```diff
@@ -508,6 +508,11 @@ def _get_cached_node(self, node_id):
 
         return self._get_node(node_id)
 
+    def update_node(self, node_ids):
+        """Update cwa config file automatically"""
```

Comment:
This may be extended over time to do more batch node updates beyond just cloudwatch, so I would recommend omitting this doc string.